app/javascript/react/components/Navbar/Navbar.style.tsx

```diff
@@ -39,8 +47,24 @@ const DesktopContainer = styled.div`
     justify-content: space-between;
     align-items: center;
     width: 100%;
+    padding: 2rem;
+    flex-wrap: wrap;
+    gap: 1rem;
+    flex-direction: flex-reverse;
   }
 `;
+
+const SearchContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 2px 2px 4px 0px rgba(76, 86, 96, 0.1);
```

Comment:
color for shadow from colors struct please, please skip if not possible to change